### PR TITLE
Enhancement review allow more ext burnin and naming

### PR DESCRIPTION
**Pull request by kaamaurice**
## Brief description
Enhancement for Blender Review

## Description
- Extract Playblast and thumbnail manage more file extensions.
- Blender host added to the Extract_Burnin feature.
- Review naming include not only task name but subset name like "Main" (usefull with scene with several subsets and reviews)

### CODE REVIEW

File: openpype/hosts/blender/api/capture.py [hunk 1]
```diff
@@ -76,15 +77,17 @@ def capture(
     frame_range = (start_frame, end_frame, step_frame)
 
     if filename is None:
-        filename = scene.render.filepath
+        filename = os.path.splitext(scene.render.filepath)[0]
```
Comment (Tilix4) [inline openpype/hosts/blender/api/capture.py:80]: Use `pathlib`

File: openpype/hosts/blender/api/capture.py [hunk 2]
```diff
@@ -76,15 +77,17 @@ def capture(
...
 
     render_options = {
-        "filepath": "{}.".format(filename.rstrip(".")),
+        "filepath": filename.lstrip(".") + ".",
```
Comment (Tilix4) [inline openpype/hosts/blender/api/capture.py:83]: Use `pathlib`

File: openpype/hosts/blender/api/capture.py [hunk 3]
```diff
@@ -107,7 +110,7 @@ def capture(
                 view_context=True,
             )
 
-    return filename
+    return filename + get_extension(image_settings)
```
Comment (Tilix4) [inline openpype/hosts/blender/api/capture.py:113]: Use `pathlib`

File: openpype/hosts/blender/api/capture.py [hunk 4]
```diff
@@ -123,6 +126,42 @@ def capture(
...
+        "QUICKTIME": ".mov",
+        "MPEG4": ".mp4",
+        "MPEG2": ".mpg",
+        "MPEG1": ".mpg",
+        "FLASH": ".flv",
+    },
+}
+
+def get_extension(image_settings):
+    file_format = image_settings.get("file_format")
```
Comment (Tilix4) [inline openpype/hosts/blender/api/capture.py:148]: missing docstring

File: openpype/hosts/blender/plugins/publish/extract_playblast.py [hunk 5]
```diff
@@ -98,13 +98,14 @@ def process(self, instance):
             path = capture(**preset)
 
         self.log.debug(f"playblast path {path}")
+        ext = os.path.splitext(path)[1].lstrip(".")
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/publish/extract_playblast.py:101]: Use `pathlib`

File: openpype/hosts/blender/plugins/publish/extract_thumbnail.py [hunk 6]
```diff
@@ -109,7 +109,8 @@ def process(self, instance):
         with maintained_time():
             path = capture(**preset)
 
-        thumbnail = os.path.basename(self._fix_output_path(path))
+        thumbnail = self._output_path(path)
+        ext = os.path.splitext(thumbnail)[1].lstrip(".")
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/publish/extract_thumbnail.py:113]: Use `pathlib`

File: openpype/hosts/blender/plugins/publish/extract_thumbnail.py [hunk 7]
```diff
@@ -147,10 +148,11 @@ def _fix_output_path(self, filepath):
             return None
 
         if not os.path.exists(filepath):
-            files = glob.glob(f"{filepath}.*.jpg")
+            filepath, ext = os.path.splitext(filepath)
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/publish/extract_thumbnail.py:151]: Use `pathlib`

**Review by Tilix4 — CHANGES_REQUESTED** [review]

File: openpype/hosts/blender/plugins/publish/extract_thumbnail.py (hunk 7 above)
Comment (Tilix4) [reply openpype/hosts/blender/plugins/publish/extract_thumbnail.py:152]: Considering `Path` is used all along for `filepath`, you don't need this line and use `filepath.stem` and `filepath.suffix` instead.

Comment (kaamaurice) [reply openpype/hosts/blender/plugins/publish/extract_thumbnail.py]: Ok `Path `all along look good with `Path.glob`

File: openpype/hosts/blender/api/capture.py [hunk 8]
```diff
@@ -107,7 +115,7 @@ def capture(
                 view_context=True,
             )
 
-    return filename
+    return str(filepath) + get_extension_from_image_settings(image_settings)
```
Comment (Tilix4) [inline openpype/hosts/blender/api/capture.py:118]:
```suggestion
    return filepath.with_suffix(get_extension_from_image_settings(image_settings)).as_posix()
```

File: openpype/hosts/blender/api/capture.py [hunk 9]
```diff
@@ -123,6 +131,51 @@ def capture(
...
+    "FFMcPEG": {
+        "QUICKTIME": ".mov",
+        "MPEG4": ".mp4",
+        "MPEG2": ".mpg",
+        "MPEG1": ".mpg",
+        "FLASH": ".flv",
+    },
+}
+
+def get_extension_from_image_settings(image_settings):
```
Comment (Tilix4) [inline openpype/hosts/blender/api/capture.py:152]:
```suggestion
def get_extension_from_image_settings(image_settings:dict)->str:
```

File: openpype/hosts/blender/plugins/publish/extract_playblast.py [hunk 10]
```diff
@@ -98,13 +98,14 @@ def process(self, instance):
             path = capture(**preset)
 
         self.log.debug(f"playblast path {path}")
+        ext = Path(path).suffix.lstrip(".")
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/publish/extract_playblast.py:101]: This is okay but I think `Path(path).suffix[1:]` is faster.

File: openpype/hosts/blender/api/capture.py (hunk 8 above)
Comment (kaamaurice) [reply openpype/hosts/blender/api/capture.py:118]: Line too long

Comment (Tilix4) [reply openpype/hosts/blender/api/capture.py:118]: black will solve that

Comment (kaamaurice) [reply openpype/hosts/blender/api/capture.py:118]: Yes, I will fix this within my own commit.

File: openpype/hosts/blender/api/capture.py (hunk 9 above)
Comment (kaamaurice) [reply openpype/hosts/blender/api/capture.py:152]: Arf okay, I have no typing here, I will update the entire code, docstrings, and comments.

**Pull request merged**